pom.xml

```diff
@@ -248,6 +248,35 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.ec4j.maven</groupId>
+					<artifactId>editorconfig-maven-plugin</artifactId>
+					<version>0.1.3						                        						                        						<!-- check the latest on http://search.maven.org/maven2/org/ec4j/maven/editorconfig-maven-plugin/ --></version>
+					<executions>
+						<execution>
+							<id>check</id>
+							<phase>verify</phase>
+							<goals>
+								<goal>check</goal>
+							</goals>
+						</execution>
+					</executions>
+					<configuration>
+						<!-- See http://ec4j.github.io/editorconfig-maven-plugin/ for full configuration reference -->
+						<excludes>
+							<!-- Note that maven submodule directories and many non-source file patterns are excluded by default -->
+							<!-- see https://github.com/ec4j/editorconfig-linters/blob/master/editorconfig-lint-api/src/main/java/org/ec4j/lint/api/Constants.java#L37 -->
+							<!-- You can exclude further files from processing: -->
+							<exclude>src/main/**/*.whatever</exclude>
```

Comment:
This is just a template and should be adopted to our needs, e.g. ignoring binary files.

Reply:
Not sure if we still need to configurate this more properly. The template is deleted.